health/notifications/health_alarm_notify.conf

```diff
@@ -212,8 +212,26 @@ DEFAULT_RECIPIENT_EMAIL="root"
 # to corelate messages and also will cause most clients to group all the
 # messages together.  This is enabled by default, uncomment the line
 # below if you want to disable it.
-#EMAIL_THREADING="NO"
+#EMAIL_THREADING="YES"
 
+#------------------------------------------------------------------------------
+# hangouts (google hangouts chat) global notification options
+
+# multiple recipients can be given like this:
+#                  "CHANNEL1 CHANNEL2 ..."
+
+# enable/disable sending hangouts notifications
+SEND_HANGOUTS="YES"
+
+# Login to hangouts and create an incoming webhook. You need only one for all
+# your netdata servers (or you can have one for each of your netdata).
+# Without it, netdata cannot send hangouts notifications.
+HANGOUTS_WEBHOOK_URL=""
+
+# if a role's recipients are not configured, a notification will be send to
+# this hangouts channel (empty = do not send a notification for unconfigured
+# roles):
+DEFAULT_RECIPIENT_HANGOUTS=""
```

Comment:
Some people never look at the actual documentation and just look at the config file comments, so it would be good to document in the comment right above here that a recipient is a channel in Hangouts Chat.

Reply:
Like this?
```
#DEFAULT_RECIPIENT_HANGOUTS="<Name of the channel>"
DEFAULT_RECIPIENT_HANGOUTS=""
```

Reply:
Something like that should be fine.